reference-adapter/mgramseva-ifix-adapter/src/main/java/org/egov/ifix/models/fiscalEvent/FiscalEvent.java

```diff
@@ -0,0 +1,106 @@
+package org.egov.ifix.models.fiscalEvent;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.egov.common.contract.AuditDetails;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class FiscalEvent {
```

Comment:
If you want to add DTO suffix with all the data transfer files, then follow the pattern for all the files. I think this file is also for the purpose of Data transfer. If not, ignore this comment.

Reply:
It involved with business.